remix_exercises/Challenge2.sol

```diff
@@ -0,0 +1,21 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+interface Challenge2 {
+    function callMe(uint) external;
+}
+
+contract Challenge2 {
+    event Winner(address);
+    public uint secret;
+
+    constructor(uint _secret) {
```

Comment:
do you intend to have them deploy this on goerli? if this is a local blockchain i think it seems too easy to deploy `Challenge2` with a constructor value of "10" and then just put 10 as the arg for the `callMe` function

Reply:
(they know what they deployed the contract with so they dont have to read the public `secret` variable